adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/Adapter.java

```diff
@@ -37,7 +37,7 @@ public DataBlob executeJob(AdapterConfig config) throws IllegalArgumentException
     try {
       var rawData = importer.fetch(config.protocolConfig.parameters);
       var result = interpreter.interpret(rawData, config.formatConfig.parameters);
-      return dataBlobRepository.save(new DataBlob(result.toString()));
+      return dataBlobRepository.save(new DataBlob(result));
```

Comment:
Before your change, a DataBlob was always a stringified JSON object. Now it can be any string if the `RAW` interpreter is used. The DataBlob string is also present in the published import success event. Therefore all consumer of this event must now be able to handle any string and do not make the assumption that the string is a stringified JSON object. 

According to the dependency graph, the pipeline and the notification service are consuming the import success event. Have you checked what happens in those services if the `RAW` interpreter is used?

Reply:
I don't really understand what you mean. A `DataBlob` Object was and still is, marshalled as a JSON object.
Are you talking about the `data` attribute of the `DataBlob`? This was a JSON before and can be any string now. 
This feature was requested by Martin for the data engineering workbench. 
I think it makes sense to provide a raw representation of the imported data to be able to choose how to interpret it later.

Are you worried about consumers not being able to interpret the `data` field of the `DataBlob`? I would argue that we can assume that, when consumers request a `RAW` import, they should be able to handle non-JSON data, but I don't know if I got your comment right...

Reply:
> Are you talking about the `data` attribute of the `DataBlob`?

Yes exactly. Sorry, what I was not clear enough.

> Are you worried about consumers not being able to interpret the `data` field of the `DataBlob`?

Yes exactly, especially the pipeline service, because I think it is currently assuming that the `data` field is a stringified JSON.
And we should not assume anything from our users, especially from a fault tolerance perspective. They can create pipelines for `RAW` interpreted data and therefore we must make sure the pipeline service does at least not crash because of that.

Reply:
Hmm, I think what Jonas detected is a little gap in our domain.
For the adapter (/preview) we need the RAW interpreter, and for the datasource (-> triggering pipelines) we actually do not (yet).. That means we should still ensure that only JSON is sent to downstream services. 
Maybe an alternative implementation approach makes sense, where the AdapterEndpoint checks if a Format is specified, and if not the raw data is returned? Or even add another endpoint for that? (/dataImport/raw)

Reply:
Okay, I think I get what you mean now. I agree that we should ensure fault tolerance but maybe there is a misunderstanding here:

when triggering a data import via the ``/dataImport`` endpoint, we do not publish an amqp event. This only happens at the ``/datasource/{id}/trigger`` endpoint so the pipeline service is not affected by this. 
 
Does this resolve your concerns or did I misunderstand you?

Reply:
I think the shorter solution would be to add a second endpoint to just do the import without interpretation ;)

Reply:
I decided to check for RAW format in the datasource creation and update endpoints.
I think this is the shortest and cleanest solution because adding another endpoint would have implied a lot of redundant code or fundamental refactoring.